app/components/groups/creating-groups/add-group-service.js

```diff
@@ -0,0 +1,42 @@
+;(function () {
+  angular
+    .module("todo")
+    .service("CreatingGroupService", CreatingGroupService)
+
+  CreatingGroupService.$inject = ["GroupResource", "FileService", "SessionService"]
+
+  function CreatingGroupService(GroupResource, FileService, SessionService) {
+    var self = this;
+    self.group = {
+      title: "",
+      items: [],
+      view: false,
+      users: [],
+      files: []
```

Comment:
I think you don't use it..
for me it's better to use it instead of `self.filesToUpload`